modules/apps/27-interchain-accounts/controller/ibc_module.go

```diff
@@ -42,13 +42,13 @@ func (im IBCModule) OnChanOpenInit(
 	chanCap *capabilitytypes.Capability,
 	counterparty channeltypes.Counterparty,
 	version string,
-) error {
+) (string, error) {
 	if !im.keeper.IsControllerEnabled(ctx) {
-		return types.ErrControllerSubModuleDisabled
+		return "", types.ErrControllerSubModuleDisabled
 	}
 
 	if err := im.keeper.OnChanOpenInit(ctx, order, connectionHops, portID, channelID, chanCap, counterparty, version); err != nil {
-		return err
+		return "", err
 	}
 
 	// call underlying app's OnChanOpenInit callback with the appVersion
```

Comment:
we should be returning the version passed in, not the underlying application version

Reply:
Are you suggesting changing the comment? 

We are passing the version the same as before.

Reply:
updated the comment anyway :+1: